app/models/claim/transfer_detail.rb

```diff
@@ -16,7 +16,7 @@ class TransferDetail < ApplicationRecord
     include TransferBrainDelegatable
 
     belongs_to :claim, class_name: 'Claim::TransferClaim', inverse_of: :transfer_detail
-    acts_as_gov_uk_date :transfer_date, error_clash_behaviour: :override_with_gov_uk_date_field_error
+    acts_as_gov_uk_date error_clash_behaviour: :override_with_gov_uk_date_field_error
```

Comment:
As `:transfer_date` is not longer using `gov_uk_date`, is this line still required?

Reply:
Not required. Removed in latest push